open-hackathon-server/src/hackathon/storage/azure_storage.py

```diff
@@ -22,13 +22,128 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 """
+__author__ = 'ZGQ'
+
 from storage import Storage
+import json
+from uuid import uuid1
+from time import strftime
+from flask import g
+
+
+from hackathon import RequiredFeature
+from hackathon.constants import FILE_TYPE, HEALTH_STATUS, HEALTH
+
 
 __all__ = ["AzureStorage"]
 
 
 class AzureStorage(Storage):
-    """Hackathon file storage that saves all templates on MS azure"""
+    """Hackathon file storage that saves all templates on Azure storage
+
+    template files will be save at "<hostname>/<container_name>/<blob_name>"
+    uploaded images will be save at "<hostname>/<container_name>/<blob_name>"
```

Comment:
Can you replace both `<container_name>` with the default values defined in config.py? And please update the format to an valid azure url like `"http://azure_storage_url/container/<blob_name>"` where `azure_storage_url` and `container` should be the right value